pages/wg/dei.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Diversity, Equity, & Inclusion Working Group
+subtitle:
+permalink: /wg/dei/
+---
+
+# Overview
+
+The US-RSE Association is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community in the US. The US-RSE Diversity, Equity, and Inclusion (DEI) working group is for those who are interested in regular discussions regarding DEI in the RSE community. The goal of the DEI working group is to promote the values of the US-RSE DEI Mission Statement with a particular focus on promoting opportunities for DEI education to the wider community. The group meets bi-weekly via Zoom and welcomes all newcomers.
+
+To get involved with the DEI working group, visit the `#dei-discussion` channel on the US-RSE slack, or contact the <a href="mailto:dei_wg@us-rse.org">DEI working group list</a>.
```

Comment:
I forget how to do this, but I do remember someone figured out how to do a link to a specific channel. Could we do that here?

Reply:
Done!